backend/inventario_servicio/gestor_proveedores/infraestructura/file_downloader.py

```
@@ -0,0 +1,15 @@
+import logging
+from io import StringIO
+
+logging.basicConfig(level=logging.ERROR)
+logger = logging.getLogger(__name__)
+
+def download_file_from_url(url):
+    try:
+        response = requests.get(url)
+        response.raise_for_status()
+        return StringIO(response.text)
+    except requests.RequestException as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)